CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ include_directories(${RAJA_INCLUDE_DIRS})
 
 set(RAJA_PERFSUITE_VERSION_MAJOR 0)
 set(RAJA_PERFSUITE_VERSION_MINOR 2)
-set(RAJA_PERFSUITE_VERSION_PATCHLEVEL 0)
+set(RAJA_PERFSUITE_VERSION_PATCHLEVEL 2)
```

Comment:
Why is this going from 0 to 2? Shouldn't this be 0.2.1?

Reply:
I forgot to update it last time.

Reply:
👍